arcon_allocator/src/lib.rs

```diff
@@ -110,6 +120,13 @@ impl Allocator {
         self.allocations
             .insert((self.alloc_epoch, id), (mem, layout));
 
+        #[cfg(feature = "allocator_metrics")]
+        {
+            increment_counter!("alloc_counter", "arcon_allocator" => "arcon_allocator");
+            gauge!("bytes_remaining",self.bytes_remaining() as f64 ,"arcon_allocator" => "arcon_allocator");
+            gauge!("total_bytes",self.limit as f64 ,"arcon_allocator" => "arcon_allocator");
```

Comment:
For now the total_bytes will always be the same as the initial limit in the ``Allocator::new(..)`` so let's move the gauge call up there.